CMakeLists.txt

```diff
@@ -170,8 +170,11 @@ if("${CMAKE_CXX_COMPILER_ID}" STREQUAL "Clang" OR
 
   # OpenSSL is deprecated on later versions of Mac OS X. The long-term solution
   # is to provide a CommonCryto implementation.
-  if (APPLE AND CASS_USE_OPENSSL)
-    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wno-deprecated-declarations")
+  if (APPLE)
+    set(CMAKE_MACOSX_RPATH 1)
```

Comment:
Without this `cmake` logs warning:
```
CMake Warning (dev):
  Policy CMP0042 is not set: MACOSX_RPATH is enabled by default.  Run "cmake
  --help-policy CMP0042" for policy details.  Use the cmake_policy command to
  set the policy and suppress this warning.

  MACOSX_RPATH is not specified for the following targets:

   cassandra

This warning is for project developers.  Use -Wno-dev to suppress it.
```

Reply:
Looks like this was added in the cmake version (2.8.12) we moved to as a minimum with the last release.  I'm hardly an expert on builds on MacOS but this seems pretty reasonable to me based on what I've read.